tasks/eth/022-holocene-fp-upgrade/.env

```diff
@@ -8,3 +8,6 @@ SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=8  # noop
 SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=11 # +1, task 021
 
 SIMULATE_WITHOUT_LEDGER=0 # 1
+
+# we change the WETH in this task, which is non-standard
+DISPUTE_GAME_CHANGE_WETH=true
```

Comment:
Do we need this flag for `base-003-holocene-fp-upgrade` ?

Reply:
No, because Base correctly _didn't_ change their DelayedWETH. The default during an upgrade is to not change it, but if it does need to change, setting this overwrite allows it.